### ISSUE TITLE
device_only_remembered_on_user_prompt fix in complete example

### ISSUE THREAD

**Issue by akshaykrjain** [mineiros-io/terraform-aws-cognito-user-pool#28]
```
Error: Unsupported argument

  on cognito.tf line 34, in module "cognito_user_pool":
  34:   device_only_remembered_on_user_prompt = false

An argument named "device_only_remembered_on_user_prompt" is not expected
here.
```

Seems like this variable is removed but _**complete**_ example still have this.

**Comment by mariux** [comment]
@akshaykrjain thanks for reporting it. nice catch. fixed in #29

**Comment by mariux** [comment]
updated example released as 0.4.1 should also now be available on the registry
